upgrade-notes/backend_20250206_212223.md

```diff
@@ -0,0 +1,13 @@
+#### add product recalculation queue deduplication ([#3669](https://github.com/shopsys/shopsys/pull/3669))
+
+- ensure you have Redis on version 7.4 (see #3673)
+- method use `\Shopsys\FrameworkBundle\Model\Product\ProductFacade::iterateAllProductIds()` was removed, use `iterateAllProductIdsExceptVariant()`instead
+- method use `\Shopsys\FrameworkBundle\Model\Product\ProductRepository::iterateAllProductIds()` was removed, use `iterateAllProductIdsExceptVariant()`instead
+- `ProductRecalculationMessage` messages no longer contain `exportScopes`. Scopes are now stored in Redis.
+- `Shopsys\FrameworkBundle\Model\Product\Recalculation\ProductRecalculationRepository`
+    - method `getIdsToRecalculate()` was removed
+    - method `getMainVariantIdsOfVariants()` was removed
+    - method `dropVariantIdsFromProductIds()` was removed
+    - method `getVariantIds()` was removed
+    - use `replaceVariantIdsWithMainVariantIds()` or `getIdsToRecalculateByMainVariantIds()` instead
+- see #project-base-diff to update your project
```

Comment:
maybe it is worth mentioning the docs as well? `docs/asynchronous-processing/product-recalculations.md`

Reply:
good idea - 0f39d5d369 (I have also fixed docs building error due to non-existing navigation item)